MAINTAINERS.yml

```diff
@@ -2818,6 +2818,17 @@ STM32 Platforms:
     STM32 SOCs, dts files and related drivers. ST nucleo, disco and eval
     boards.
 
+Enclustra Platforms:
+  status: maintained
+  maintainers:
+    - fkokosinski
+  collaborators:
+    - tgorochowik
+  files:
+    - boards/arm/mercury_xu/
```

Comment:
I see one board here, we do not add areas for single boards, is this area going to grow beyond this 1 single board?

Reply:
Hi, thanks for your comment.

> is this area going to grow beyond this 1 single board

No, we don't plan to add more boards in the near future as far as I'm aware.

> I see one board here, we do not add areas for single boards

Understood. For context: this change was inspired by https://github.com/zephyrproject-rtos/zephyr/pull/65968, which revealed that this board is currently not covered by any of the areas of maintenance. This is because the ARM Arch area, unlike other Arch areas in Zephyr, doesn't cover all of its boards.

This means that the following condition from https://docs.zephyrproject.org/latest/project/project_roles.html#maintainers-file is not met without taking a manual action:

> All submitted pull requests should have an assignee

Could you guide me what would be the best approach to properly address this issue, and to get this board maintained? Thanks

Reply:
@nashif What do you think would be a good solution here? @tgorochowik and @fkokosinski are willing to maintain this platform, but otherwise it should end up somewhere else (we're fine with it as long as someone is maintaining it), not as it currently is (effectively unmaintained).